pylint_odoo/misc.py

```diff
@@ -172,6 +178,62 @@ def _check_{NAME_KEY}(self, module_path)
                         node.file = node_file_original
                         node.lineno = node_lineno_original
 
+    def visit_module(self, node):
+        self.wrapper_visit_module(node)
+
+    def add_message(self, msg_id, line=None, node=None, args=None,
+                    confidence=UNDEFINED):
+        version = (self.manifest_dict.get('version')
+                   if isinstance(self.manifest_dict, dict) else '0')
+        version = LooseVersion(version)
+        match = self.formatversion(version.vstring)
+        if not match:
+            valid_odoo_versions = self.linter._all_options[
+                'valid_odoo_versions'].config.valid_odoo_versions
+            version = LooseVersion(valid_odoo_versions[0] if
+                                   len(valid_odoo_versions) == 1 else '0')
+            match = self.formatversion(version.vstring)
```

Comment:
If you have a valid_odoo_version of length 1 so you have a short_version here
You don't need a re.match here again